src/display/canvas.js

```diff
@@ -706,7 +706,8 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
   CanvasGraphics.prototype = {
 
     beginDrawing: function CanvasGraphics_beginDrawing(transform, viewport,
```

Comment:
I'm assuming that @yurydelendik meant that you should change the method signature like this (and also change `api.js` as indicated elsewhere):
```javascript
beginDrawing({ transform, viewport, transparency,
               backgroundColor = null, }) {
```